packages/webhooks/utils/validation.spec.js

```diff
@@ -0,0 +1,106 @@
+const { EVENT_TYPE_PATTERN, validateEventType } = require('./validation')
+
+describe('validateEventType', () => {
+    describe('valid event types', () => {
+        const validCases = [
+            'payment.created',
+            'payment.status.changed',
+            'invoice.paid',
+            'user.deleted',
+            'order.item.added',
+            'a.b',
+            'resource1.action2',
+            'my-resource.my-action',
+            'payment.status-change.completed',
+            'Payment.Created',
+            'test.event.7v5619fr',
```

Comment:
we should not allow dynamic values, its better to remove this from examples

Reply:
Enum (static 'invoice.paid' + dynamic webhookedModels.map(modelName => ['modelName.created', 'modelName.updated'])

(String + validation)

Reply:
Refactored